homeassistant/components/sensor/sytadin.py

```diff
@@ -0,0 +1,83 @@
+"""
+Support for Sytadin Traffic, French Traffic Supervision.
+
+Systadin website : http://www.sytadin.fr
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sytadin/
+"""
+import logging
+
+from homeassistant.const import LENGTH_KILOMETERS
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['lxml', 'requests']
```

Comment:
`requests` is already a core requirement. Perhaps `xmltodict` could simplify the handling of the XML.